src/seedu/addressbook/data/person/Unit.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Unit {
+	private final String _unit;

```

Comment:
good to see you use explicit notation to differentiate between private members and parameters. :)